test/integration/dataunion/withdraw.test.ts

```diff
@@ -44,14 +50,16 @@ const testWithdraw = async (
     const tx1 = await tokenMainnet.mint(adminWalletMainnet.address, parseEther('100'))
     await tx1.wait()
 
-    const adminClient = new StreamrClient(config.clientOptions as any)
+    const adminClient = new StreamrClient(config.clientOptions)
 
     const dataUnion = await adminClient.deployDataUnion()
     const secret = await dataUnion.createSecret('test secret')
     log(`DataUnion ${dataUnion.getAddress()} is ready to roll`)
     // dataUnion = await adminClient.getDataUnionContract({dataUnion: "0xd778CfA9BB1d5F36E42526B2BAFD07B74b4066c0"})
 
-    const memberWallet = new Wallet(`0x100000000000000000000000000000000000000012300000001${Date.now()}`, providerSidechain)
+    testWalletId += 1
+    const memberWallet = new Wallet(`0x100000000000000000000000000000000000000012300000000000001${testWalletId}`, providerSidechain)
```

Comment:
Hm, I do wonder if it'd avoid possible state leaking between tests if the addresses were always random? e.g. 

```js
`0x${crypto.randomBytes(32).toString('hex')}`
```

Reply:
sure, though I think there is value in deterministic addresses too

Reply:
I mean, technically those numbers/keys/addresses are "state" but it's never used except as an opaque object, and it only needs to be unique. Random addresses would work just as well, that was the idea of the Date.now() pretty much. But I'll keep that randomBytes thing in mind.